README.md

```diff
@@ -65,11 +74,13 @@ $encrypted = $evervault->encrypt([
 $result = $evervault->run('hello-cage', $encrypted);
 ```
 
-## API Reference
+## Reference
+
+At present, there are two functions available in the PHP SDK: `$evervault->encrypt()` and `$evervault->run()`.
```

Comment:
```suggestion
```

Reply:
went with: "The Evervault PHP SDK exposes two functions."